host-configs/alcf-builds/corona_sycl.cmake

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
ALCF directory is for scripts to use on Argonne machines. Corona is an LC machine. So it should go in the LC builds directory and have toss4 in the script name. Also, please update copyright header following the format of the others.